test/integration/integration_test.go

```diff
@@ -279,31 +288,57 @@ func checkConfigKey(configPath string, condition string, keyPath string) error {
 
 func compareExpectedWithActualContains(expected string, actual string) error {
 	if !strings.Contains(actual, expected) {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
```

Comment:
I do not think it is necessary to prematurely optimize. this one extra call internally is not going to kill us, and especially not in a test situation. consistency and readability are more important.

Besides, most of the optimizations are done by the compiler... never be the compiler yourself unless performance is really an issue.